src/Widgets/EventMenu.vala

```diff
@@ -29,7 +29,8 @@ public class Maya.EventMenu : Gtk.Menu {
 
     construct {
         E.Source src = comp.get_data ("source");
-        bool sensitive = src.writable == true && Model.CalendarModel.get_default ().calclient_is_readonly (src) == false;
+        bool can_modify_cal = Model.CalendarModel.get_default ().calclient_is_readonly (src) == false;
```

Comment:
Suggest use `bool can_modify_cal = !Model.CalendarModel.get_default ().calclient_is_readonly (src);`

Reply:
Of course. "==false" is plain stupid. Will change before anyone sees it (I keep flagging stuff like that in code reviews all the time.)